silo/api.py

```diff
@@ -155,10 +158,13 @@ def create(self, request, *args, **kwargs):
                 as e:
             return Response(e, status=status.HTTP_400_BAD_REQUEST)
 
+        url_subpath = '/activity/forms/uuid/{}'.format(form_uuid)
+        form_url = urljoin(settings.ACTIVITY_URL, url_subpath)
         read = Read.objects.create(
             owner=tola_user.user,
             type=ReadType.objects.get(read_type='CustomForm'),
             read_name=read_name,
+            read_url=form_url
```

Comment:
Whoever had the "great" idea to store the URLs in the database instead of generating them dynamically... 💩If we decide to change the domain or the path it would break all URLs

Reply:
I don't know. It's how the frontend of the Track works now.